components/studentPortal/lessons/UnitView.tsx

```diff
@@ -4,39 +4,65 @@ import UnitNodeView from "./UnitNodeView";
 
 export type UnitViewProps = {
   data: Unit;
+  userRole: "freemium" | "student" | "coach";
 };
 
-export const UnitView: React.FC<UnitViewProps> = ({ data }: UnitViewProps) => {
+//These are hardcoded lists of the Units/Nodes that will be included with the Freemium version
+const freemiumUnits = ["Introduction", "Github", "TailwindCSS", "React"];
+const freemiumNodes = [
```

Comment:
The unit view component should only have to know how to render itself. It shouldn't have to know what units make up the freemium curriculum.

The curriculum info is part of our business' logic. We need to separate that from UI component code.

Reply:
I created a freemiumUnits.ts file that includes only the units that we want to include with freemium and includes a few empty units to gray out.  This can be easily changed and does not affect the regular user.